src/index.js

```diff
@@ -0,0 +1,46 @@
+import './style.css';
+import './logo.png';
+import {
+  fetchCategoryMeals, displayMeals, transferLikes, likesCountUpdate,
+} from './modules/renderList.js';
+import { updateCategoryCounts } from './modules/mealCounter.js';
+
+const menuLinks = document.querySelectorAll('header a');
+const containerMeals = document.getElementById('list');
+
+const renderMealCategory = async (category) => {
+  const categoryMeals = await fetchCategoryMeals(category);
+  displayMeals(categoryMeals);
+};
+
+const setupHeaderLinkListeners = () => {
+  menuLinks.forEach((link) => {
+    const { dataset: { category } } = link;
+    link.addEventListener('click', async (e) => {
+      e.preventDefault();
+      await renderMealCategory(category);
+    });
+  });
```

Comment:
- [ ] You still have an issue with the comments, when trying to add a comment the app is crushing and the comments are repeated on all the cards. Kindly fix the issue.